addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3635,7 +3634,19 @@ msgctxt "#844"
 msgid "Deactivate"
 msgstr ""
 
-#empty strings from id 845 to 849
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+msgctxt "#845"
+msgid "Are you sure you want to delete this repeating timer, including all timers it has scheduled?"
```

Comment:
@jjd-uk - at least to me this reads a bit odd. I was never a big fan of "repeating timer" when it's actually no timer but more of a "timer rule" (I'm sure there is a better wording for it). IMO it's easier to understand if it would sound something like "Are you sure you want to delete this timer rule and all it's scheduled timers?"